test/basedriver/helpers-specs.js

```diff
@@ -45,4 +46,27 @@ describe('helpers', function () {
       forceWindows.restore();
     });
   });
+
+  describe('#renameKey', function () {
```

Comment:
would be good to also have a kitchen-sink test that mixes deeply-embedded keys and primitives in various fashions